ui/analyse/src/view.ts

```diff
@@ -93,11 +93,29 @@ function makeConcealOf(ctrl: AnalyseCtrl): ConcealOf | undefined {
 }
 
 function renderAnalyse(ctrl: AnalyseCtrl, concealOf?: ConcealOf) {
+  const showNextChapter = ctrl.study?.nextChapter() && !ctrl.study?.members.canContribute() && !ctrl.practice;
```

Comment:
why `!ctrl.study?.members.canContribute()`?

Reply:
The idea was that if you're creating a study, you tend to always be on the last move which would show the full-blown attention-seeking button even though you probably don't want to go to the next chapter. You're just adding more moves.

Could still be useful to them though since ofc many people also study their own studies. Maybe just always showing it muted or when not in "rec" mode or something would be better 🤷‍♂️

Reply:
(just cleaned up my commit history, no changes since you looked at this last)

Yeah, interesting ideas @benediktwerner . I kind of like showing it when not in rec mode, which is really what we were trying to get at with this condition anyways.

Reply:
Just checking out what it's like if we were to do the above and only show it when NOT in REC mode. The only weird thing I saw was here, when you're not in record mode and looking at an interactive chapter. Rather specific case, but the clutter here caught my eye:
![Screenshot 2022-02-01T14-39-45 - Google Chrome](https://user-images.githubusercontent.com/12655228/152063587-3091f980-46b6-4445-9bb5-0edf32cce192.png)